modules/common/util/map.go

```diff
@@ -99,3 +102,20 @@ func MergeMaps[K comparable, V any](baseMap map[K]V, extraMaps ...map[K]V) map[K
 
 	return mergedMap
 }
+
+// GetStringListFromMap - It returns a list of strings based on a comma
+// separated list assigned to the map key. This is usually invoked to normalize
+// annotation fields where a list of items is expressed with a comma separated
+// list of strings.
+// Example:
+// input: in["additionalSubjectNamesKey"] = "foo.bar,bar.svc,*.foo.bar"
+// output: [foo.bar bar.svc *.foo.bar]
+func GetStringListFromMap(in map[string]string, key string) []string {
```

Comment:
This seems what we discussed in the previous iteration. Let's agree on this and then I update the other patches.

Reply:
yes, thats what I had in mind of doing a common func, instead of adding only for this particular use case func.